packages/go_router/lib/src/router.dart

```diff
@@ -295,9 +295,6 @@ class GoRouter implements RouterConfig<RouteMatchList> {
   @override
   late final GoRouteInformationParser routeInformationParser;
 
-  /// Returns `true` if there is at least two or more route can be pop.
-  bool canPop() => routerDelegate.canPop();
```

Comment:
why was this removed while pop is still there?

Reply:
you are right, I keep thinking this as a state that should be remove, but it is actually needed for pop as well.

Reply:
added it back and updated migration guide

Reply:
@chunhtai Also consider correcting CHANGELOG.md for v9.0.0

Reply:
done